### PR TITLE
Edit config using getenv

### PR DESCRIPTION
**Pull request by rudibroekhuizen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,38 @@ A very simple phppgadmin docker image that uses the latest official release (7.1
 ## Usage
 
 ```
-docker run -p 8080:8080 andersevenrud/phppgadmin:latest
+docker run -p 8080:8080 naturalis/phppgadmin:latest
```
Comment (andersevenrud) [inline README.md]: Why was this changed ?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+version: "3"
```
Comment (andersevenrud) [inline docker-compose.yml]: What is the purpose of having a docker-compose file in this repository ? :)

Comment (rudibroekhuizen) [reply docker-compose.yml]: Removed the file

File: README.md (hunk 1 above)
Comment (rudibroekhuizen) [reply README.md]: Changed back

**Comment by andersevenrud** [comment]
Closing because this repo is deprecated
